python/mit/binding.py.in

```diff
@@ -167,12 +195,12 @@ libmit.mit_destroy.argtypes = [c_void_p]
 libmit.mit_register_args.argtypes = [c_void_p, c_int, c_void_p]
 
 libmit.mit_align.restype = c_uword
-libmit.mit_align.argtypes = [c_uword]
+libmit.mit_align.argtypes = [c_uword, c_uint]
 
-libmit.mit_is_aligned.argtypes = [c_uword]
+libmit.mit_is_aligned.argtypes = [c_uword, c_uint]
 
 def align(addr):
-    return libmit.mit_align(addr, size_word)
+    return int(libmit.mit_align(addr, size_word))
```

Comment:
This call to `int()` could be removed if `mit_align()` already returns an int.

Reply:
See above.